src/runtime/web/index.js

```diff
@@ -1,6 +1,35 @@
-import { createSimpleContext } from './api';
-import { View } from '../../bindings/html/view';
+import { prioritizeLocales } from '../../intl/index';
+import { HTMLLocalization } from '../../bindings/html';
+import { documentReady, getResourceLinks, getMeta } from './util';
```

Comment:
The difference is that `getResourceLinks` and `getMeta` is not bindings-specific, but runtime-specific. This allows you to have HTML bindings in a Service Worker runtime for instance which won't have `getMeta` at all.  In this PR I'm making it clearer (hopefully!) that the bindings are about *how* things are localized, and runtime is about plugging the localization in and initializing it.  Language negotiation is the first step to initialize the `Localization` object and thus I thought it would be fitting to put it in `runtime/`.